server/server.js

```diff
@@ -68,15 +74,29 @@ app.use(bodyParser.json({limit: '50mb'}));
 app.use(amazonSNSConfirmMiddleware.bind(null, amazonConfig));
 app.use(amazonSNSNotificationMiddleware.bind(null, amazonConfig));
 
-if (!process.env.DISABLE_IMAGE_SCALING_QUEUE) {
-  // Using require to make the dependencies optional.
-  app.set('imageQueue', require('./image.queue')(app, redisConfig.host, redisConfig.port));
-}
-else {
-  app.set('imageQueue', {
-    add: () => {} // if the imageQueue is disabled, we simply supress created jobs.
-  });
-}
+app.startQueues = () => {
+  if (!process.env.DISABLE_IMAGE_SCALING_QUEUE) {
```

Comment:
It is a bit confusing that DISABLE_IMAGE_SCALING_QUEUE is used to disable all queues